frontend/src/metabase-types/store/state.ts

```diff
@@ -31,7 +30,7 @@ export interface State {
   qb: QueryBuilderState;
   requests: RequestsState;
   routing: RouterState;
-  settings: SettingsState;
+  [Api.reducerPath]: typeof Api.reducer; // FIXME how to type Api slice?
```

Comment:
@sloansparger i'm stumped on this one

Reply:
I think you want `ReturnType<typeof Api.reducer>` ([link to the friendly manual](https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-state-type)). We should probably just do something like `export type ApiState = ReturnType<typeof Api.reducer>;` from `frontend/src/metabase/api/api.ts`.